libraries/resource_hollandbackup_install.rb

```diff
@@ -0,0 +1,19 @@
+require 'chef/resource/lwrp_base'
+
+class Chef
+  class Resource
+    class HollandbackupInstall < Chef::Resource::LWRPBase
+      self.resource_name = :hollandbackup_install
+      actions :create, :delete
+      default_action :create
+
+      attribute :repo_name, kind_of: String, default: nil
```

Comment:
I think the default is already nil on these if you don't supply one. Ditto for all the other resources defined.